uber/decorators.py

```diff
@@ -30,6 +30,15 @@ def with_check(*args, **kwargs):
     return with_check
 
 
+def redirect_if_at_con_to_kiosk(func):
+    @wraps(func)
+    def with_check(*args, **kwargs):
+        if c.AT_THE_CON:
+            raise HTTPRedirect(c.KIOSK_REDIRECT_URL)
+        return func(*args, **kwargs)
+    return with_check
+
+
 def check_if_can_reg(func):
```

Comment:
would actually love to make something like:

```redirect_if(c.AT_THE_CON, c.KIOSK_REDIRECT_URL)```

but don't have time at the moment